export default function handleError(error) {
  logger.log("error", "Error happened", { error });
}
